spec/log_weasel/resque_scheduler_spec.rb

```diff
@@ -50,6 +56,20 @@
         end
       end
 
+      context "in debug mode" do
```

Comment:
Do we also need a simple assertion in a parent context (i.e. not debug mode) that `puts` is not called?

Reply:
good catch, rohit!  +1

Reply:
Why I think I can make that happen!